tools/download_swagger_ui_3.R

```diff
@@ -6,7 +6,7 @@ library(rvest)
 
 local({
 
```

Comment:
If the files are exactly the same except for the version... Can we make this a `for` loop to avoid file duplication?

Reply:
I don't understand this comment. I interpret it as do not duplicate file within dist5, dist4, dist3? Where would you put the for loop you are hinting at.

Reply:
```suggestion
  for (swagger_ui_version in c("3.52.5", "4.19.1", "5.10.0")) {

```

And:
* Indent all the code inside the for-loop
* Close the for-loop at the end.

Reply:
> I interpret it as do not duplicate file within dist5, dist4, dist3?

Correct. If the files are the same except for the version, then we should combine them into a single update file.

Reply:
They changed the way they inform the index file between version 3 and 4. Let me sit on this one. There are probably clever ways to reduce the size of the package.

Reply:
Ah, I see. I'd be happy even if it's just combining v4 and v5. No need to make this a perfect solution.

Reply:
Sat long enough